src/context.h

```diff
@@ -131,6 +131,8 @@ typedef struct getdns_upstream {
 	/* How is this upstream doing over UDP? */
 	int                      to_retry;
 	int                      back_off;
+	size_t                   udp_responses;
+	size_t                   udp_timeouts;
```

Comment:
Since udp_responses and udp_timeouts are only  read when DAEMON_DEBUG is defined and 1, why not wrap the definition, initializations and updates in `#defined(DAEMON_DEBUG) && DAEMON_DEBUG` ... `#endif` too?